### DISCUSSION TITLE
Add a tab to see deleted devices

### DISCUSSION THREAD

**Discussion by Seluj78** [shellhub-io/shellhub#3499] [Ideas]
I would love to be able to see and list deleted devices from the UI

**Comment by henrybarreto** [comment]
Hi, @Seluj78 

We appreciate your commitment to helping ShellHub to improve.

In the community version, we don't save the deleted device, so it isn't possible to create this kind of tab. It only exists in the Cloud version due to business needs.

**Reply by Seluj78** [reply]
Alright thank you very much, I wasn't aware this was a `cloud` only feature. It does make sense ! I was wondering what all those tags in the API were for. Thanks!
